### PR TITLE
metrics model repo structure proposal

### PR DESCRIPTION
**Pull request by eyehwan**

### CODE REVIEW

**Comment by eyehwan** [comment]
@wangliang-cs Thanks Liang, I like it. I will create a patch to add it.

File: context-areas/ecosystem.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
+# Ecosystem
```
Comment (sgoggins) [inline context-areas/ecosystem.md:1]: This seems like a good addition to me, and provides some clarity vis a vi a useful example.

File: metrics-model-libs/community-activity/definition/community-activity.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
+# Community Activity
```
Comment (sgoggins) [inline metrics-model-libs/community-activity/definition/community-activity.md:1]: A really good metrics model definition.  Thanks, @eyehwan !

**Review by sgoggins — APPROVED** [review]

**Comment by sgoggins** [comment]
@klumb  & @germonprez : I think this PR does a nice job of exemplifying a structure. If you agree, one of you can merge it. I would merge it now, but I want your impressions first.

**Pull request merged**